IncludeToolbox/IncludeFormatter/IncludeLineInfo.cs

```diff
@@ -92,6 +92,8 @@ public static IncludeLineInfo[] ParseIncludes(string text, bool removeEmptyLines
                 if (multiLineCommentStart > -1 && multiLineCommentStart < includeOccurence) // Multi line comment started in same line, but before #include.
                     continue;
 
+                if (lines[line].Contains(".inl>") || lines[line].Contains(".inl\"")) // Don't remove inline files.
```

Comment:
Halfway in, I just realized that this feature is already in. Woops.
Going to put a meaningful default in there and test it again.
Options>Include Toolbox>Try and Error Include Removal